intermittent failure in rpc_invalidateblock.py

Seen in doc-only PR: https://cirrus-ci.com/task/4697486185988096?logs=ci


```
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 133, in main
    self.run_test()
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/rpc_invalidateblock.py", line 90, in run_test
    assert_equal(chain_tips, self.nodes[1].getchaintips())
  File "/tmp/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/util.py", line 56, in assert_equal
    raise AssertionError("not(%s)" % " == ".join(str(arg) for arg in (thing1, thing2) + args))
AssertionError: not([{'height': 24, 'hash': '1e8c7b54ab7c4556c4602fc89c0796d104260df2e2f8d2de9006d2446676d71c', 'branchlen': 0, 'status': 'active'}, {'height': 6, 'hash': '46727213ae639f43edd7fde6d5c2dcd54cc182f7c3a5c7a652784a7ece5aa0cf', 'branchlen': 2, 'status': 'invalid'}, {'height': 4, 'hash': '0fefbfc08f4e5e4ae4dd1a74113c580530bda9bfa85ed2c40e59ee8ff5bc352d', 'branchlen': 4, 'status': 'headers-only'}] == [{'height': 24, 'hash': '1e8c7b54ab7c4556c4602fc89c0796d104260df2e2f8d2de9006d2446676d71c', 'branchlen': 0, 'status': 'active'}, {'height': 6, 'hash': '46727213ae639f43edd7fde6d5c2dcd54cc182f7c3a5c7a652784a7ece5aa0cf', 'branchlen': 2, 'status': 'invalid'}, {'height': 4, 'hash': '0fefbfc08f4e5e4ae4dd1a74113c580530bda9bfa85ed2c40e59ee8ff5bc352d', 'branchlen': 4, 'status': 'headers-only'}, {'height': 3, 'hash': '4ff28db3f1e2007c2a8f321dc6759ffdb7ff0c81ea9334d982049b9c8ae674a9', 'branchlen': 1, 'status': 'headers-only'}])
```

--- a/test/functional/rpc_invalidateblock.py
+++ b/test/functional/rpc_invalidateblock.py
@@ -85,9 +85,8 @@ class InvalidateTest(BitcoinTestFramework):
         assert_equal(self.nodes[1].getbestblockhash(), blocks[-1])
 
         self.log.info("Verify that invalidating an unknown block throws an error")
-        chain_tips = self.nodes[1].getchaintips()
         assert_raises_rpc_error(-5, "Block not found", self.nodes[1].invalidateblock, "00" * 32)
-        assert_equal(chain_tips, self.nodes[1].getchaintips())
+        assert_equal(self.nodes[1].getbestblockhash(), blocks[-1])
 
 
 if __name__ == '__main__':

